pandas/core/computation/eval.py

```diff
@@ -12,7 +12,8 @@
 from pandas.util._validators import validate_bool_kwarg
 
 from pandas.core.computation.engines import _engines
-from pandas.core.computation.expr import Expr, _parsers, tokenize_string
+from pandas.core.computation.expr import Expr, _parsers
```

Comment:
we should de-privatize parsers (and change it to a more informative, parser_engines, but should be a followup)

Reply:
#30681